bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat(ingress): stop creating the "mk-" ingress controller as this is now created by fleetshard

…



## Description


PS: Do not merge this until all long living kafkas have been migrated to use the new ingress controller

## Verification Steps

Using changes in this patch
- Verify that we can create a Kafka and it moves to ready state when `EnableKafkaExternalCertificate` is `false` (no need to check when `true` because no code path has changed on this)
- Once the Kafka moves to `ready` verify that we can produce/consume messages

## Checklist (Definition of Done)
